cmake/modules/FindDEAL_II_KOKKOS.cmake

```diff
@@ -162,4 +165,34 @@ if(KOKKOS_FOUND)
     # warning #940-D: missing return statement at end of non-void function
     enable_if_supported(DEAL_II_CXX_FLAGS "-Xcudafe --diag_suppress=940")
   endif()
+
+  #
+  # Extract version numbers:
+  #
+  if(NOT KOKKOS_VERSION)
+    if(Kokkos_VERSION)
+      set(KOKKOS_VERSION "${Kokkos_VERSION}")
+    else()
+      message(FATAL_ERROR "FindPackage(Kokkos) did not set KOKKOS_VERSION!")
+    endif()
+  endif()
```

Comment:
```suggestion
```

The current logic ensures that `KOKKOS_VERSION` is always populated. So this is not needed any more.